wunderio/client-UK-FSA-beta
# FSA-992: Implement autoscroll to multi-step forms

To test:
- Go to a contact form with multiple steps (./contact, then pick one of those)
- Once you reach step 2 and later, the form should automatically scroll into view
